mlir/lib/Dialect/Rock/Transforms/AlignTiling.cpp

```diff
@@ -111,8 +103,9 @@ Value makeRegs(PatternRewriter &b, MemRefType::Builder &mrb, Location loc,
                                        srcMemType.getElementType())));
 }
 
-static Value applyTransforms(PatternRewriter &b, ThreadwiseWriteAllOp storeOp,
-                             Value src) {
+static Value
+applyTransforms(PatternRewriter &b, ThreadwiseWriteAllOp storeOp, Value src,
+                const SmallVector<TransformMapAttr> &relativeViewsOnStore) {
```

Comment:
`const SmallVector<T> &` is spelled `ArrayRef<T>`

Reply:
ah yes... my bad